configs/period_thephpleague.json

```diff
@@ -1,25 +1,28 @@
 {
   "index_name": "period_thephpleague",
   "start_urls": [
-    "https://period.thephpleague.com"
+    {
+      "url": "http://period.thephpleague.com/(?P<version>.*?)/",
+      "variables": {
+        "version": ["4.0", "3.0"]
```

Comment:
Please update your meta data from your website instead.

For example, https://period.thephpleague.com/3.0/ is having a 4.0 meta

Reply:
thanks I've updated the website accordingly ... about the selectors can I also configure them on the website because I failed to find that information on the API documentation ?

Reply:
Thanks. I have update your index.

Please remove the variables and we can merge this PR